src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Vault/SetAzureRmRecoveryServicesBackupProperties.cs

```diff
@@ -50,11 +50,9 @@ public override void ExecuteCmdlet()
             {
                 if (this.BackupStorageRedundancy.HasValue)
                 {
-                    UpdateVaultStorageTypeRequest vaultStorageRequest = new UpdateVaultStorageTypeRequest();
```

Comment:
Also, per the previous cmdlet, this needs to support SHouldProcess, and when you update the output, add an OutputType attribute with the appropriate OutputType